src/backend/crud/agent.py

```diff
@@ -17,9 +17,14 @@ def create_agent(db: Session, agent: Agent) -> Agent:
     Returns:
       Agent: Created agent.
     """
-    db.add(agent)
-    db.commit()
-    db.refresh(agent)
+    try:
```

Comment:
Suggestion: how about making a decorator that wraps every crud transaction in a try:except?

Reply:
🤯

Reply:
I always forget decorators exist hahah